backend/db/schema.sql

```diff
@@ -0,0 +1,85 @@
+CREATE TABLE "Users" (
+	"userId" serial PRIMARY KEY,
+	"username" varchar(120) NOT NULL UNIQUE,
+	"email" varchar(255) NOT NULL UNIQUE,
+	"fullName" varchar(255),
+	"avatarPath" varchar(255),
+	"location" varchar(80),
+	"company" varchar(120),
+	"website" varchar(255),
+	"bio" text,
+	"isAdmin" boolean DEFAULT FALSE,
+	"createdAt" timestamp DEFAULT CURRENT_TIMESTAMP,
+	"updatedAt" timestamp DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE "AuthProviders" (
+	"providerId" smallserial PRIMARY KEY,
+	"providerName" varchar(60) NOT NULL UNIQUE
+);
+
+CREATE TABLE "Auths" (
+	"authId" serial PRIMARY KEY,
+	"userId" int NOT NULL REFERENCES "Users" ON DELETE CASCADE,
+	"authProviderId" smallint NOT NULL REFERENCES "AuthProviders" ON DELETE CASCADE,
+	"password" varchar(100)
+);
+
+CREATE TABLE "RefreshSessions" (
+	"tokenId" bigserial PRIMARY KEY,
+	"userId" int NOT NULL REFERENCES "Users" ON DELETE CASCADE,
+	"refreshToken" uuid NOT NULL UNIQUE,
+	"userAgent" varchar(255) NOT NULL,
+	"fingerprint" varchar(200) NOT NULL,
+	"expiresIn" bigint NOT NULL,
+	"createdAt" timestamp with time zone NOT NULL DEFAULT now()
```

Comment:
here we have `timestamp with time zone`, in other places `timestamp` - they both are correct, but not mix of them